src/cfml/system/util/CompileDSL.cfc

```diff
@@ -0,0 +1,88 @@
+
+/**
+*********************************************************************************
+* Copyright Since 2014 CommandBox by Ortus Solutions, Corp
+* www.coldbox.org | www.ortussolutions.com
+********************************************************************************
+* @file  D:\Dev\Repos\commandbox\src\cfml\system\util\CompileDSL.cfc
+* @author  Balbino Aylagas
+* @description
+*
+* I am a helper object for executing compile actions for "javac" command and
+* jarring actions for "jar" command Create me and call my
+* methods to compile and jar any java project, then .run() will execute me.
+*
+*/
+
+component accessors=true {
+
+    property name='command';
+    property name='target';
+    property name='projectRoot';
+    property name='fileSystemUtil';
+	property name='wirebox';
```

Comment:
This doesn't belong here, does it?  Wirebox would be an injected property, which would go under the "DI" section and require an `inject` attribute.